### ISSUE TITLE
Can't reproduce the paper's MUGE result

### ISSUE THREAD

**Issue by yiranxie233** [OFA-Sys/Chinese-CLIP#396]
您好，感谢您开源Chinese-CLIP！
我在MUGE采用单张3090进行复现，发现最终结果远低于您论文中所写，请问是我的实验超参数出了什么问题吗，下面是我设置的超参数:
~~~cmd
--nproc_per_node=1
--nnodes=1
--node_rank=0
--master_addr=localhost
--master_port=8514
Chinese-CLIP-master/cn_clip/training/main.py
--train-data
Chinese-CLIP-master/dataset/MUGE/lmdb/train
--val-data
Chinese-CLIP-master/dataset/MUGE/lmdb/valid
--resume
Chinese-CLIP-master/pretrained_weights/clip_cn_vit-b-16.pt
--reset-data-offset
--reset-optimizer
--logs
Chinese-CLIP-master/experiments/
--name
muge_finetune_vit-b-16_roberta-base_bs128_1gpu
--save-step-frequency
999999
--save-epoch-frequency
1
--log-interval
1
--report-training-batch-acc
--context-length
52
--warmup
100
--batch-size
128
--valid-batch-size
128
--valid-step-interval
150
--valid-epoch-interval
1
--accum-freq
1
--lr
5e-5
--wd
0.001
--max-epochs
20
--vision-model
ViT-B-16
--use-augment
--text-model
RoBERTa-wwm-ext-base-chinese
~~~
我自己修改了在运行过程中途测试验证集的代码，使其直接测试对应的rank值，发现刚开始学习时文找图rank下降非常大最终得到的epoch_last的结果为{Image2Text R@1: 22.83 | Image2Text R@5: 52.52 | Image2Text R@10: 65.17 | Text2Image R@1: 41.75 | Text2Image R@5: 71.57 | Text2Image R@10: 81.51 | logit_scale: 4.526 | }同时附上我的训练日志。

[out_2025-08-12-03-47-17.log](https://github.com/user-attachments/files/21751902/out_2025-08-12-03-47-17.log)

请问这种情况如何解决呢？
